platform-dom/src/main/java/com/softicar/platform/dom/engine/IDomEngine.java

```diff
@@ -105,6 +105,7 @@ public interface IDomEngine {
 	void setPreventDefaultBehavior(IDomNode node, DomEventType type, boolean enabled);
 
 	void setPreventDefaultOnMouseDown(IDomNode node, boolean enabled);
+//	void setPreventDefaultOnWheel(IDomNode node, Set<DomModifierKey> modifiers);
```

Comment:
you shall not pass

Reply:
#480 is now merged, btw

Reply:
removed line and now calling it with CTRL, changed zooming to CTRL+WHEEL